### PR TITLE
doc: add min deposit amounts

### PR DESCRIPTION
**Pull request by ggutoski**

### CODE REVIEW

File: resources/testnet-releases.md [hunk 1]
```diff
@@ -34,3 +34,11 @@ Core Version  | Start Height | End Height
 ------------- | ------------- | -------------
 v0.10.7 | 0 | 14700
 v0.13.0 | 14701 | N/A
+
+# Minimum deposit amounts
```
Comment (milapsheth) [inline resources/testnet-releases.md:38]: Maybe say by destination chains?

File: resources/testnet-releases.md [hunk 2]
```diff
@@ -34,3 +34,13 @@ Core Version  | Start Height | End Height
 ------------- | ------------- | -------------
 v0.10.7 | 0 | 14700
 v0.13.0 | 14701 | N/A
+
+# Minimum transfer amounts
+
+For each asset X in (AXL, UST, LUNA) and each external chain Y in (Ethereum, non-Ethereum EVM, Cosmos/IBC) any transfer of asset X to/from chain Y must exceed the minimum amount given in the table below.
```
Comment (milapsheth) [inline resources/testnet-releases.md:40]:
```suggestion
For each asset X in (AXL, UST, LUNA) and each external chain Y in (Ethereum, non-Ethereum EVM, Cosmos/IBC) any transfer of asset X to chain Y must exceed the minimum amount given in the table below.

Note: If a smaller amount is deposited, it'll be stuck in the queue until another deposit is made to the same deposit address bringing the total above the minimum.
```

Comment (ggutoski) [reply resources/testnet-releases.md]: added more clarification

**Review by milapsheth — APPROVED** [review]
Looks great, thanks!

**Pull request merged**